subiquity/server/ubuntu_drivers.py

```diff
@@ -262,4 +389,14 @@ def get_ubuntu_drivers_interface(app) -> UbuntuDriversInterface:
         else:
             cls = UbuntuDriversHasDriversInterface
 
+    if "subiquity-fake-pci-devices" in app.opts.kernel_cmdline:
+        log.debug("Using umockdev wrapper")
+        cls = UbuntuDriversFakePCIDevicesInterface
+
+    # For quickly testing MOK enrollment we install on server and force no gpgpu
+    # The caveat to this is that it also has to be an online install
+    if "subiquity-server-force-no-gpgpu" in app.opts.kernel_cmdline:
+        log.debug("Forcing no gpgpu drivers. Requires online install on server.")
+        is_server = False
```

Comment:
would you rename the `is_server` variable? `gpgpu` makes more sense now I think.